server/src/test/scala/manifests/paxinfo/FlightManifestSummaryFromManifestSpec.scala

```diff
@@ -106,11 +105,11 @@ class FlightManifestSummaryFromManifestSpec extends Specification {
   }
 
   "When deserializing age ranges we should get back the correct age range " >> {
```

Comment:
I know this is an old test, but it's not really clear what it's doing. It's not actually checking the serialisation & deserialisation. It would be good to change it to actually serialise and deserialise using upickle